src/babel-preset/common.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  presets: [
+    require.resolve('babel-preset-react'),
+  ],
+  plugins: [
+    require.resolve('babel-plugin-add-module-exports'),
+    // stage 3
+    require.resolve('babel-plugin-transform-async-generator-functions'),
+    require.resolve('babel-plugin-transform-object-rest-spread'),
+    // stage 2
+    require.resolve('babel-plugin-transform-class-properties'),
```

Comment:
I know this is copy-pasted, just thought I'd make sure we add a comment here: the class properties proposal was merged with the private fields proposal into the ["class fields" proposal](https://github.com/tc39/proposal-class-fields), which is currently stage 3. I don't see a specific Babel transform for that proposal yet, though, but there is https://github.com/babel/proposals/issues/12 which tracks this proposal. Maybe add a link to that Babel proposal?

Reply:
Yeah, that's a good point. I imagine once there is a solid class fields proposal it will go into babel-preset-env and collide with this transform, but I shall leave a comment nonetheless